harness/tests/experiment/utils.py

```diff
@@ -413,7 +413,13 @@ def create_trial_instance(trial_def: Type[det.Trial]) -> None:
     with tempfile.TemporaryDirectory() as td:
         trial_instance = experimental.create_trial_instance(
             trial_def=trial_def,
-            config={"hyperparameters": {"global_batch_size": det.Constant(16)}},
+            config={
```

Comment:
What is this change for?

Reply:
Unfortunately, we break this test by creating model and optimizers in the constructor, which requires these hparams. We used not to create them in the test.